indexd/index/blueprint.py

```diff
@@ -780,6 +780,11 @@ def handle_auth_error(err):
     return flask.jsonify(error=str(err)), 403
 
 
+@blueprint.errorhandler(AuthzError)
+def handle_authz_error(err):
+    return flask.jsonify(error=str(err)), 401
+
```

Comment:
What it was thinking was instead of it being an AuthZ error you can make it some general arborist error where you can pass the error message + error code to this function and do something like `return flask.jsonify(error=err_msg), err_code`

Reply:
I think the decision to make it an auth error was because regardless of it was a failure of the arborist service or a token expiration, the authorization process still failed